src/routes/areas/createArea.test.ts

```diff
@@ -11,6 +11,22 @@ describe('POST /areas', () => {
     server = createServer({ authRoute: [createArea] as RouteOptions[] });
   });
 
+  it('returns a 200 status code when creates an area correctly', async () => {
+    const response = await server.inject({
```

Comment:
We must verify if the new area has already been created in the DB. Please also validate that in this test.